triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -322,6 +331,109 @@ final class Converter(
       "config" -> config,
     )
 
+  def fromCommand(command: Command): SValue = command match {
```

Comment:
As call sites will simply throw converter exceptions, I've intentionally not attempted to work with Either's here